backend/lib/utils.js

```diff
@@ -182,7 +182,8 @@ module.exports = {
     const { method } = extrinsic.toHuman().method;
     const args = JSON.stringify(extrinsic.args);
     const hash = extrinsic.hash.toHex();
-    const doc = extrinsic.meta.docs.toString().replace(/'/g, "''");
+    // const doc = extrinsic.meta.docs.toString().replace(/'/g, "''");
+    const doc = '';
```

Comment:
Why hardcode?

Reply:
Meta doc is not available in the extrinsic. So I have assigned an empty string to it.